src/main/java/cz/idomatojde/entity/Offer.java

```diff
@@ -0,0 +1,138 @@
+package cz.idomatojde.entity;
+
+import com.sun.istack.NotNull;
+
+import javax.persistence.*;
+import java.sql.Date;
+import java.time.LocalDateTime;
+
+
+/*
+Created by Jiri Vrbka
+ */
+@Entity
+public class Offer {
+
+    @Id
+    @GeneratedValue(strategy= GenerationType.IDENTITY)
+    private Long id;
+
+    @NotNull
+    @ManyToOne
+    private User owner;
+
+    @NotNull
+    private String title;
+
+    @NotNull
+    private String description;
+
+    private int price;
+
+    @NotNull
+    @Temporal(TemporalType.TIMESTAMP)
+    private Date createdDate;
+
+    @NotNull
+    @Temporal(TemporalType.DATE)
+    private Date expirationDate;
+
+    @OneToMany
+    private TimetableEntry event;
+
+    private Category category;
+
+    private Integer capacity;
+
+    private Integer registered;
```

Comment:
What is the reasoning behing this being an "Integer" and price being "int"?

Reply:
Inspired from reference project